<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleAdjust</class>
 <widget class="QWidget" name="SimpleAdjust">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_num_1">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>190</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_num_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>190</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_num_3">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>190</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_num_clr">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>450</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>CE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_num_4">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>270</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_num_6">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>270</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_num_0">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>450</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_num_5">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>270</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_num_7">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>360</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_num_9">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>360</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_num_Point">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>450</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_num_8">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>360</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_simple">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>311</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>单 点 校 准</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_max">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>240</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_current">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>430</y>
     <width>240</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_min">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>340</y>
     <width>240</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_except">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>520</y>
     <width>240</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>151</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>最大值</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>151</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>最小值</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>380</y>
     <width>151</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>当前值</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>480</y>
     <width>151</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>期望值</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>610</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>700</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/image/bg_05.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <zorder>btn_ok_2</zorder>
  </widget>
  <widget class="QPushButton" name="btn_ok_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>610</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>校准</string>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>btn_num_1</zorder>
  <zorder>btn_num_2</zorder>
  <zorder>btn_num_3</zorder>
  <zorder>btn_num_clr</zorder>
  <zorder>btn_num_4</zorder>
  <zorder>btn_num_6</zorder>
  <zorder>btn_num_0</zorder>
  <zorder>btn_num_5</zorder>
  <zorder>btn_num_7</zorder>
  <zorder>btn_num_9</zorder>
  <zorder>btn_num_Point</zorder>
  <zorder>btn_num_8</zorder>
  <zorder>label_simple</zorder>
  <zorder>lineEdit_max</zorder>
  <zorder>lineEdit_current</zorder>
  <zorder>lineEdit_min</zorder>
  <zorder>lineEdit_except</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>btn_ok</zorder>
  <zorder>btn_ok_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
